ui/docs/tutorial/debug-task.md

```diff
@@ -39,7 +39,7 @@ VNC connection to a running task execution environment.
 
 There are two ways to access this mode. The easiest is to enter the task's ID
 in the [task inspector](https://tools.taskcluster.net/groups). Under
-"actions", select "create interactive task". If you have permission to create
+"actions", select "Create with SSH task". If you have permission to create
```

Comment:
The Interactive connect also allows a user to connect via VNC. Let's rename "SSH" to "SSH/VNC".